cfep-03.md

```diff
@@ -0,0 +1,47 @@
+
+<table>
+<tr><td> Title </td><td> Providing manually uploaded builds on the conda-forge anaconda channel</td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Hadrien Mary &lt;hadrien.mary@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sep 27, 2016</td></tr>
+<tr><td> Updated </td><td> Oct 17, 2019</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Some packages, for example Qt and gcc, are very hard to build on the currently used CIs (CircleCI, Travis, AppVeyor, and AzurePipelines) due to time limitation.
+
+While this is a small set of packages we need a protocol to upload builds on conda-forge anaconda channel that are built off-CIs (local builds).
+
+Note that this is a "last resort" option and we should always try to build on CIs first!
+
+## Specification
+
+The protocol we propose is:
+
+- The package needs a feedstock and will be maintained via PRs as any other package;
+
+- If the CI time limits are the only impediment for the build one should ping the core team in the feedstock and a local build;
```

Comment:
```suggestion
- If the CI resources (which includes time, memory, special hardware) are the only impediments for the build, one should ping the core team in the feedstock and a local build;
```

Reply:
Funny. I cannot merge that from here"

![Screenshot from 2019-11-14 14-33-25](https://user-images.githubusercontent.com/950575/68881412-dc389b80-06eb-11ea-8e50-ff40fadba15b.png)